jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -516,7 +516,8 @@ private void registerRestoreEnvVariables() {
 
 						List<String> illegalKeys = new ArrayList<>(0);
 						while ((entry = envFileReader.readLine()) != null) {
-							if (!entry.isBlank()) {
+							entry = entry.trim();
```

Comment:
some env vars may have trailing white space which needs to be respected

Reply:
Sure, changed to keep the leading or trailing spaces of entry string.